### ISSUE TITLE
`elm-xref` doesn't work when dependency has not been installed with 0.19.0 specifically

### ISSUE THREAD

**Issue by jfmengels** [zwilias/elm-xref#8]
I noticed `elm-xref` crashes when a dependency has not been installed with the specific 0.19.0 version.
When I have a project in 0.19.1, and I have never installed a dependency using Elm 0.19.0 (or never used 0.19.0), `elm-xref` will print an error like

```
{ Error: ENOENT: no such file or directory, scandir '/home/jeroen/.elm/0.19.0/package/elm/core'
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/home/jeroen/.elm/0.19.0/package/elm/core' }
```

and then not do any verification. The process then exits with status code 0, meaning that it doesn't block our tests or CI (where we never installed dependencies with 0.19.0), but we also "silently" lost the guarantee of not having unused functions.

## To reproduce

- Go to your Elm home (`~/.elm/` on UNIX systems and `C:/Users/<user>/AppData/Roaming/elm` on Windows), and (re)move the `0.19.0` directory if present
- Create a new Elm project using Elm 0.19.1
- Run `elm-xref` on the project
- Notice a crash

## Investigation

`elm-xref` seems to have [hardcoded looking for dependencies in the `0.19.0` directory](https://github.com/zwilias/elm-xref/blob/master/lib/runner.js#L290).

For applications, I suggest looking for the version present in the project's `elm.json` file.
For packages, I am not sure what would be best, as we don't have the version of the compiler that is actually used (0.19.1 could be used by the developer, but the package supports 0.19.0). Maybe trying all `0.19.x` folders present in the Elm home from latest to oldest until a dependency is found?

## Thanks!

While I'm here, let me take the opportunity to thank you for this wonderful tool :blush: 
It has been so useful to me and my projects to keep them clean :heart:

**Comment by zwilias** [comment]
Released in v4.1.0!

**Comment by jfmengels** [comment]
Awesome, thanks a lot :blush: 

It seems to work just fine :+1:
